package/pom.xml

```diff
@@ -207,34 +185,46 @@
             </goals>
             <configuration>
               <excludeTransitive>true</excludeTransitive>
-              <useBaseVersion>true</useBaseVersion>
-              <outputDirectory>${distro.microfrontendsDir}</outputDirectory>
-              <includeArtifactIds>microfrontends</includeArtifactIds>
+              <outputDirectory>${distro.spaDir}</outputDirectory>
+              <includeArtifactIds>spa</includeArtifactIds>
             </configuration>
           </execution>
-          <!-- Microfrontends config -->
+          <!-- SPA config -->
           <execution>
-            <id>unpack-microfrontends-config-refapp</id>
+            <id>unpack-refapp-config-spa</id>
             <phase>package</phase>
             <goals>
               <goal>unpack-dependencies</goal>
             </goals>
             <configuration>
               <excludeTransitive>true</excludeTransitive>
-              <useBaseVersion>true</useBaseVersion>
-              <outputDirectory>${distro.microfrontendsConfigDir}</outputDirectory>
-              <includeArtifactIds>microfrontends-config-refapp</includeArtifactIds>
+              <outputDirectory>${distro.spaConfigDir}</outputDirectory>
+              <includeArtifactIds>spa-config-refapp</includeArtifactIds>
+            </configuration>
+          </execution>
+          <!-- Unpack OpenMRS distro package -->
+          <execution>
+            <id>unpack-openmrs-distro-package</id>
+            <!-- pre-integration-test is immediately after the package phase
+                 ideally, this would be done in a post-package phase -->
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${openmrsRunDistroTargetDir}</outputDirectory>
+              <artifact>${project.groupId}:openmrs-distro-package:${project.version}:zip</artifact>
             </configuration>
           </execution>
         </executions>
       </plugin>
 
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-antrun-plugin</artifactId>
-        <version>1.7</version>
         <executions>
           <execution>
-            <id>rename-jars-to-omod</id>
+            <id>rename jars to omod</id>
```

Comment:
We shouldn't need to rename jars to omod, or rename the war file.  We should be using the sdk plugin based on the openmrs-distro.properties file to pull down the war, modules, and owas that are defined and rename them appropriately.